src/scenario/ScenarioRepository.cpp

```diff
@@ -24,6 +24,7 @@
 #include "../core/Path.hpp"
 #include "../core/String.hpp"
 #include "../PlatformEnvironment.h"
+#include "../core/Util.hpp"
```

Comment:
right, I originally thought it was `../util/Util.hpp`, that's how it ended here.